docs/tiger_data_spec.md

```diff
@@ -133,3 +136,113 @@ A typical search will return records in the following format when using ElasticS
 
 }
 ```
+
+## Data creation
+
+The census geocoder uses Elasticsearch synonyms to resolve abbreviations (i.e. St for Street, or MD for Maryland).
+The synonyms.txt file with the synonyms definition must be installed in every node in the cluster, in the same directory as the elasticsearch.yml configuration file.
+
+* First, create the index, with the synonyms analyzer settings, and apply that analyzer to the corresponding fields:
```

Comment:
@jmarin after our conversation today I'm going to merge this as is. Then I'll come back and make a few updates to show updating the mappings.